scilpy/io/utils.py

```diff
@@ -174,11 +183,17 @@ def assert_outputs_exist(parser, args, required, optional=None):
     """
     Assert that all outputs don't exist or that if they exist, -f was used.
     If not, print parser's usage and exit.
-    :param parser: argparse.ArgumentParser object
-    :param args: argparse namespace
-    :param required: string or list of paths
-    :param optional: string or list of paths.
-                     Each element will be ignored if None
+
+    Parameters
+    ----------
+    parser: argparse.ArgumentParser object
+        Parser.
+    args: list
+        Argument list.
+    required: string or list of paths
```

Comment:
Same comment here for required and optional

Reply:
Same as above. I limited myself to changing the docstring syntax and/or adding the type where necessary without changing the variable names. Let me know what you would propose as an accurate naming of the variables.

Reply:
Done in 4a5df3b.